aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/CleanupBucketIntegrationTests.java

```diff
@@ -61,7 +61,11 @@ public void testCleanup() {
                     || bucket.getName().startsWith("s3-low-level-presigned-url")
                     || bucket.getName().startsWith("java-bucket-policy-integ-test")
                     || bucket.getName().startsWith("copy-object-integ-test")
-                    || bucket.getName().startsWith("java-server-side-encryption-integ-test")) {
+                    || bucket.getName().startsWith("java-server-side-encryption-integ-test")
+                    || bucket.getName().startsWith("acceleration-integration-test-")
```

Comment:
The existing code construct here is absolutely batty.

At _minimum_, it should be like:
```
List<String> bucketPrefixes = Arrays.asList(
    "acceleration-integration-test-",
   ...
);
for (String prefix : bucketPrefixes) {
    if (bucket.getName().startsWith(prefix)) {
        // do stuff
    }
}
```

From there, generating the `List<String>` in a more automated way (from file, from inspection of test data, etc.) would be desirable.

@jpignata's suggestion of using an AWS Tag on the bucket like `special-test-slug-55` or whatever would also be a nice improvement.

Reply:
Updated to use a common prefix for all the test buckets.